src/test/java/co/rsk/federate/signing/hsm/advanceblockchain/HsmBookkeepingClientImplTest.java

```diff
@@ -432,6 +433,38 @@ public void resetAdvanceBlockchain_UnknownError() throws Exception {
         hsmBookkeepingClient.resetAdvanceBlockchain();
     }
 
+    @Test
+    public void getBlockchainParameters_ok() throws JsonRpcException, HSMClientException {
+        Keccak256 expectedCheckpoint = new Keccak256("0000000000000000000000000000000000000000000000000000000000000001");
+        int expectedMinimumDifficulty = 492567908;
```

Comment:
We should test with a mainnet value, accumulated difficulty for 4000 blocks. Make sure we can handle it